src/app/team/serviceForm/whoIsThisClothingFor/WhatKindOfClothingView.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Selector from '../../../../components/selector';
+import ConfirmationOptions from '../../../../components/form/ConfirmationOptions';
+
+function FormView({ value, onConfirm, onEdit }) {
+  return (
+    <div className="w-100">
+      <div style={{ fontSize: '13px', marginBottom: '1em' }} className="font-weight-bold mt-2">
+        What kind of clothing is it?
```

Comment:
Shouldn't this be `Who is the clothing for?`?
(The filenames - WhatKindOfClothingView, WhatKindOfClothingEdit - also make this confusing.)